mobile/about.html

```diff
@@ -12,6 +12,48 @@ <h1>About this document</h1>
+          <dl>
+            <dt>Exploratory work</dt>
+            <dd><ul>
+              <li>Mention High Dynamic Range and Wide Gamut Color on the Web in <a data-page="media"></a></li>
+              <li>Mention User-Agent Client Hints in <a data-page="adaptation"></a></li>
+              <li>Mention Storage buckets in <a data-page="storage"></a></li>
+              <li>Mention Storage Access API in <a data-page="storage"></a></li>
+              <li>Mention performance.measureMemory API in <a data-page="performance"></a></li>
+              <li>Mention main thread scheduling APIs in <a data-page="performance"></a></li>
+              <li>Mention requestPostAnimationFrame in <a data-page="performance"></a></li>
+              <li>Mention Web Locks API in <a data-page="storage"></a> and <a data-page="lifecycle"></a></li>
+              <li>Mention User Idle Detection in <a data-page="lifecycle"></a></li>
+              <li>Mention MiniApps in <a data-page="lifecycle"></a></li>
+            </ul></dd>
+
+            <dt>Technologies in progress</dt>
+            <dd><ul>
+              <li>Mention Constructable Stylesheet Objects in <a data-page="adaptation"></a></li>
+              <li>Mention new CSS media features in <a data-page="adaptation"></a></li>
+              <li>Mention Web App Manifest - Application Information in <a data-page="lifecycle"></a></li>
+              <li>Move Badging API to technologies in progress in <a data-page="lifecycle"></a> and <a data-page="userinput"></a></li>
+              <li>Move Web Share Target API to technologies in progress in <a data-page="lifecycle"></a></li>
+              <li>Move WebGPU to technologies in progress in <a data-page="graphics"></a></li>
+              <li>Move the Storage specification to technologies in progress in <a data-page="storage"></a></li>
+              <li>Move Open Screen Protocol to technologies in progress in <a data-page="media"></a></li>
+              <li>Rename Feature Policy to Permissions Policy in <a data-page="security"></a></li>
+            </ul></dd>
+
+            <dt>Well-deployed technologies</dt>
+            <dd><ul>
+              <li>Mention HTML/CSS Ruby in <a data-page="graphics"></a></li>
+              <li>Move main Web components specifications to well-deployed technologies in <a data-page="adaptation"></a></li>
+            </ul></dd>
+          </dl>
+          <p>The implementation info has also been updated to reflect known implementation status in September 2020. Some of these updates may have been triggered by updates to underlying platform status sites. Main updates are:</p>
+          <ul>
+            <!-- TODO: Should we mention the implementation status changes in Edge? There are many, many, changes to the implementation status in this update because of the EdgeHTML/Chakra->Blink/V8 switch. -->
```

Comment:
Yes, I think there should be just one line for Edge here to report that the implementation status is now for the new version.

(There are probably other implementation status changes that are worth reporting here on top of that)

Reply:
```suggestion
            <li>Implementation info for Edge now targets the latest version of Edge, based on Chromium. A number of features that were not supported or in development correctly appear as shipped in Edge as a result</li>
            <li>Web Components technologies are now well supported across browsers, see <a data-page="adaptation"></a></li>
            <li>Web Animations has shipped in Safari, see <a data-page="graphics"></a></li>
```